source/coding-the-legislation/35_periods.md

```diff
@@ -42,12 +38,48 @@ This [YAML test](writing_yaml_tests.md) on `income_tax` evolution over time show
       2015-01: 416.6667 # The income tax rate changes in 2015
       2016-01: 416.6667
       2017-01: 0 # The salary is not set for this period and defaults to 0
+```
+
+This syntax could also be used in Python.  
+This example sets 3 years salaries for two persons and calculates how much they get for one month:
```

Comment:
This example is more about the `divide_by_period` input option, I'm not sure it's illuminating in the context of discussing periods strictly.

Reply:
Strong 👍